pkg/kubelet/logs/container_log_manager.go

```diff
@@ -193,6 +193,11 @@ func (c *containerLogManager) Clean(containerID string) error {
 	if err != nil {
 		return fmt.Errorf("failed to get container status %q: %v", containerID, err)
 	}
+
+	if status.GetLogPath() == "" {
+		return fmt.Errorf("log path of container %s can't be empty", containerID)
```

Comment:
Though I am not sig-node member, I left a comments.

How about return nil if LogPath is empty, because logs is not exist.
Do you know what will happen if it will return nil?
I think it is better in the start process instead of clean to check if the log path is Empty or not.

Reply:
Thanks for your comments.

I think, in CRI, it should not return an empty LogPath for Kubelet in containerStatus, so this is an error.
There are no error messages in the Kubelet log if return nil, which is not friendly for developer, especially when implementing Runtime using CRI;
And if return nil , It will get containerStatus  in removeContainerLog (via CRI) and then try to delete the log legacy symbolic link of the container.I don't think we need to do that anymore.
(Although the result is expected if return nil)

Reply:
Thank you for your reply.

I thought that if Clear returns an error, then removeContainer will also return an error, and container will not be removed forever.
I understood that it is important to be able to judge the bug by the log during Runtime development. And Normally, it will not occur.
If so, I think that this PR is good.

Reply:
Certainly, Container will not be removed when Clean and removeContainer  return error step by step, cause function m.runtimeService.RemoveContainer will not be invoked  in removeContainer.